config/locales/idv/fr.yml

```diff
@@ -229,6 +229,20 @@ fr:
       wrong_address: Pas la bonne adresse?
     images:
       come_back_later: Lettre avec un crochet
+    legal_statement:
+      information_collection: >-
+        Esta recopilación de información cumple con los requisitos del título 44
```

Comment:
@dawei-nava I've been doing an audit of French/Spanish mismatches and stumbled upon this. It appears we have French and Spanish mixed up in these texts. (This text is French)

Reply:
Actually, strangely it looks like the paragraph changes to French about halfway through ("Nous estimons ..."). The same happens in the Spanish implementation (changes from French to Spanish at the burden estimation portion ("Calculamos que tomará ...")

Reply:
@aduth , thanks, not sure what happened, it was supposed to be copied from translations.